src/components/Extra.vue

```diff
@@ -617,7 +617,7 @@
     <div class="report-sidebar">
```

Comment:
while working on this, you can move `report-sidebar` to its own component, and also move the logic from `ux.js` to its script.

Reply:
Sure will migrate the code into it's own component.